requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
It is a pain but we need to generate the requirements.txt file in a 3.9 env otherwise we might lose requirements needed there. Or run `pip-compile -P msgspec requirements.in` to just touch that package